portal/celery_worker.py

```diff
@@ -39,6 +39,10 @@ def setup_periodic_tasks(sender, **kwargs):
     for job in ScheduledJob.query.all():
         task = getattr(tasks, job.task, None)
         if task:
+            if not task.func_dict.get('scheduled_task', False):
+                logger.error("Scheduled job {} does not have "
```

Comment:
I think `logging` has built-in functionality for [sprintf-style string formatting](https://docs.python.org/2/library/logging.html#logging.Logger.debug); slightly more readable